File: CSharpFundamentals/CSharpOOPbasics/InheritanceEx/MordorCruelPlan/Animals/Kitten.cs
using System;

namespace Animals
{
    public class Kitten : Cat
    {
        public Kitten(string name, int age, string gender)
            : base(name, age, gender)
        {
            this.Gender = "Female";
        }

        public override void ProduceSound()
        {
            Console.WriteLine("Meow");
        }
    }
}

File: JavaDB/DatabaseAdvancedFrameworks/JavaOOPPrinciplesEx/Animals/models/Kitten.java
package Animals.models;

import static Animals.constraints.Constraints.KITTEN_SOUND;

public class Kitten extends Cat {

    public Kitten(String name, int age, String gender) {
        super(name, age, gender);
    }

    @Override
    public String produceSound() {
        return KITTEN_SOUND;
    }
}
